raspiKernelInfo.sh

```diff
@@ -52,18 +53,24 @@ function extractStageDescription() {
 	fi
 }
 
-displayAndExec "uname -a"
-displayAndExec "dpkg --print-architecture"
-displayAndExec "getconf LONG_BIT"
+if [ "$1" == "-f" ] ; then
```

Comment:
I prefer to use `[[ ]]` or `(( ))` all the time. Would you please update this line accordingly?